template.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: backstage.io/v1alpha1
+kind: Template
+metadata:
+  name: template-expressjs
+  title: ExpressJs Application
+  description: Standard Express JS frontend with recommended configuration for HMCTS
+  tags:
+    - recommended
+    - frontend
+    - expressjs
+spec:
+  owner: Platform Engineering (#rpe)
+  templater: cookiecutter
+  type: service
+  schema:
+    required:
+      - product
+      - component_id
+      - description
+    properties:
+      product:
+        title: Product
+        type: string
+        description: Product this component belongs to, normally the team name, e.g. cmc
+      component_id:
+        title: Component name
+        type: string
+        description: Name of the component, e.g. frontend
+      description:
+        title: Description
+        type: string
+        description: Description of the component
+      http_port:
+        title: Port
+        type: integer
+        default: 8080
```

Comment:
cookie cutter default is using 3100 why different?

Reply:
copy paste :)